compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -417,7 +411,7 @@ materializeEncodingForTarget(RankedTensorType tensorType,
         return a.cast<TypeAttr>().getValue();
       }));
   SmallVector<TileMxNxK> enumeratedTileMxNxK =
-      enumerateMatmulTileMxNxK(user, elementTypes, targetAttr);
+      enumerateMatmulTileMxNxK(encoding, elementTypes, targetAttr);
```

Comment:
I think it is better to pass `cDims.value()` to the method directly. Then we don't have to get the contraction dims in all the method; we don't need to check again if they are failed or not.